FhirToDataLake/synapse-e2e-test.yml

```diff
@@ -7,137 +7,60 @@ steps:
     Write-Host $("App name: " + $appName)
     
     $databaseName = $appName
```

Comment:
Is `appName`  different for each pipeline execution?

Reply:
Yes, the **appName** was generated in [first task](https://github.com/microsoft/FHIR-Analytics-Pipelines/blob/main/FhirToDataLake/synapse-e2e-test.yml#L4) and align with each pipeline build number.

But if one pipeline execution fails, the test container and database may have been created, need to first check error/logs and delete container or database, then retry the pipeline.